e2e/provision/gce_init.sh

```diff
@@ -28,32 +28,39 @@ RUN_E2E=${NEPHIO_RUN_E2E:-$(get_metadata nephio-run-e2e "false")}
 REPO=${NEPHIO_REPO:-$(get_metadata nephio-test-infra-repo "https://github.com/nephio-project/test-infra.git")}
 BRANCH=${NEPHIO_BRANCH:-$(get_metadata nephio-test-infra-branch "main")}
 NEPHIO_USER=${NEPHIO_USER:-ubuntu}
+HOME=${HOME:-/home/$NEPHIO_USER}
+NEPHIO_REPO=${NEPHIO_REPO:-$HOME/test-infra}
```

Comment:
NEPHIO_REPO is expected to be a Git URL when passed from the command line. That is, the script supports calling it with

NEPHIO_REPO=https://github.com/my-user/nephio-test-infra.git ./gce_init.sh

I don't think we actually *do* that anywhere. But this use of NEPHIO_REPO seems to be the local clone directory, not the upstream repository.

Reply:
I would suggest naming this something like "REPO_DIR" or "LOCAL_REPO" or something.

Reply:
Good catch, I didn't notice that was used previously for other purpose.